pyiron_base/jobs/functioncontainer.py

```diff
@@ -0,0 +1,14 @@
+from pyiron_base.jobs.job.generic import GenericJob
+
+
+class FunctionContainer(GenericJob):
+    def __init__(self, project, job_name):
+        super().__init__(project, job_name)
+        self._python_only_job = True
+
+    def run_static(self):
+        for kwargs in self.input:
+            self.function(**kwargs)
```

Comment:
`GenericJob` doesn't have an `input` class, so it's not totally clear what `in` is doing here, but naively I would expect this to call `self.function` multiple times. Do you mean something like this?

```suggestion
        self.function(**self.input.to_dict())
```